ci/k8s/run-kuberay-doc-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "--- Setup k8s environment"
+SKIP_CREATE_KIND_CLUSTER=1 bash ci/k8s/prep-k8s-environment.sh
+
+echo "--- Install Python dependencies"
+pip install -c python/requirements_compiled.txt pytest nbval bash_kernel
+python -m bash_kernel.install
+pip install "ray[default]==2.43.0"
```

Comment:
It's better to use the same version as the Ray images in the Ray documentation.

Reply:
Updated to 2.41.0